Config/DefaultEngine.ini

```diff
@@ -1,7 +1,8 @@
 
 
 [/Script/EngineSettings.GameMapsSettings]
-GameDefaultMap=/Engine/Maps/Templates/OpenWorld
+GameDefaultMap=/Game/Lvl_Visualization.Lvl_Visualization
+EditorStartupMap=/Game/Lvl_Visualization.Lvl_Visualization
 
 [/Script/WindowsTargetPlatform.WindowsTargetSettings]
 DefaultGraphicsRHI=DefaultGraphicsRHI_DX12
```

Comment:
Extra line

Reply:
This is another of those config files that I don't directly edit, should I still try to add the new line?

Reply:
No worries in that case